hashicorp/boundary-ui
# feat: 🎸 create manage projects in org route and view

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-13882

# Description

Add new page "Manage Projects in Org"
Note: This PR is missing the icon being added to the "Add custom scopes" button once the scopes have been saved.

## Screenshots (if appropriate)

https://github.com/hashicorp/boundary-ui/assets/107949262/76c7aa27-7a06-49f2-bd64-a07e25bfe9e7


## How to Test

1. Go to a global level role.
2. Click on manage dropdown
3. Click on Manage Scopes
4. Click on "Add custom scopes"
5. User should now see "Manage Custom Scopes" page 
6. The table should have the "Projects Selected" Column if projects exist for that org role
7. User can click the link in the "Projects Selected" column
9. User can select and save projects.
10. Repeat steps for org level role (Note: on this level there is no "Manage Custom Scopes" page).

## Checklist
